test/service/FileTransferService.test.ts

```diff
@@ -74,9 +92,84 @@ describe('FileTransferService', () => {
 
             try {
                 await fileTransferService.upload(Buffer.from('This is a test'), 'test.not_supported');
-            } catch(err) {
+            } catch (err) {
                 expect(err.message).to.contain('Unsupported file type');
             }
         });
     });
+
+    describe('File Metadata', () => {
+
+        const fileMetadaUrl = `${URI}/${fileID}`;
+
+        it('should throw an error if the axios request fails', async () => {
+
+            createGetNockRequest(fileMetadaUrl).reply(404);
+
+            try {
+                await fileTransferService.getFileMetadata(fileID);
+            } catch (err) {
+                expect(err.constructor.name).to.eq(FileNotFoundError.name);
+            }
+
+        });
+        it('should get file metada if it exists', async () => {
+
+            createGetNockRequest(fileMetadaUrl).reply(200, expectedMetada);
+
+            await fileTransferService.getFileMetadata(fileID).then(_ => assert.deepEqual(_, expectedMetada));
```

Comment:
For consistency please extract response to variable `const fileMetadata = ...` and compare it on the next line. Please also use `chai.expect` instead of assertions` as this is our main assertion library.